### PR TITLE
Frontend codes

### PR DESCRIPTION
**Pull request by mayerm94**
use models that have aligning frontend and backend codes
Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-2034
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Bonnie Reviewer:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:** N/A

Name:

### CODE REVIEW

File: lib/cqm/converter/hds_record.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require 'execjs'
 require 'sprockets'
+require 'byebug'
```
Comment (hossenlopp) [inline lib/cqm/converter/hds_record.rb]: Remove this.
